test/e2e/provisioning/helper.go

```diff
@@ -11,55 +11,106 @@ import (
 
 var scenarios = []scenario{
 	{
-		name:                    "scenario 1 Ubuntu Docker 1.13",
-		osName:                  "ubuntu",
-		containerRuntime:        "docker",
-		containerRuntimeVersion: "1.13",
+		name:              "Ubuntu Docker Kubernetes v1.8.13",
+		osName:            "ubuntu",
+		containerRuntime:  "docker",
+		kubernetesVersion: "1.8.13",
```

Comment:
is this kubernetesVersion or only kubelet version ?

Reply:
Kubelet and kubeadm

Reply:
ok, thx.